README.md

```diff
@@ -28,6 +28,14 @@ alt="Travis CI">
 </a>
 
 
+PyMKS is an open source, pythonic implementation of the methodologies 
```

Comment:
My rewrite of that:

PyMKS is an open source, Pythonic implementation of the techniques developed under the aegis of the Materials Knowledge System (MKS) [refs]. The MKS is a methodology for building process-structure-property linkages for materials science applications. PyMKS provides efficient tools for obtaining a digital, uniform grid representation of a materials internal structure in terms of its local states, and computing hierarchical descriptors of the structure that can be used to used to build efficient machine learning based mappings to the relevant response space.

Reply:
I think that looks good! Once @auag92 finalizes his draft, I can take a look at it!